src/app/api/send/route.ts

```
@@ -0,0 +1,40 @@
+import { NextResponse } from 'next/server';
+import { Resend } from 'resend';
+import prisma from '@/utils/prisma';
+
+export async function POST(request: Request) {
+  try {
+    const { email, name } = await request.json();
+
+    const resend = new Resend(process.env.RESEND_API_KEY);
```

Let's throw an error if `process.env.RESEND_API_KEY` is not set
And also - please, add `RESEND_API_KEY` to the `.env.example`